query/src/main/scala/filodb/query/exec/PlanDispatcher.scala

```diff
@@ -15,7 +16,7 @@ import filodb.query.QueryResponse
   * from the ExecPlan.
   */
 trait PlanDispatcher extends java.io.Serializable {
-  def dispatch(plan: ExecPlan)
+  def dispatch(plan: ExecPlan, parentSpan: kamon.trace.Span)
```

Comment:
As indicated in other comments, please try to remove the span parameter.

Reply:
This is fixed